src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -241,6 +243,21 @@ public Object getAttribute(String key, Object defaultValue) {
             return defaultValue;
     }
 
+    /** returns the value as an empty list if the key was not found,
+        as a java.util.List if the value is a List or an Array,
+        as a Collections.singletonList if there is only one value */
+    @SuppressWarnings("unchecked")
+    public List<Object> getAttributeAsList(String key) {
+        Object o = getAttribute(key);
+        if( o == null) return Collections.emptyList();
+        if( o instanceof List ) return (List<Object>)o;
+        if( o.getClass().isArray() )
```

Comment:
The indents are off, there are extra spaces.  I am trying to keep the style consistent with the rest of the code.
```if (o.getClass().isArray()) {
     return Arrays.asList((Object[])o);
}```